pkg/storage/streams/store_test.go

```diff
@@ -0,0 +1,120 @@
+package streams
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/pkg/paths"
+	ranger "storj.io/storj/pkg/ranger"
+	"storj.io/storj/pkg/storage/segments"
+)
+
+var ctx = context.Background()
+
+type metaClosure = func(context.Context, paths.Path) (segments.Meta, error)
+type getClosure = func(context.Context, paths.Path) (ranger.Ranger, segments.Meta, error)
+type putClosure = func(context.Context, paths.Path, io.Reader, []byte, time.Time) (segments.Meta, error)
+type deleteClosure = func(context.Context, paths.Path) error
+type listClosure = func(context.Context, paths.Path, paths.Path, paths.Path, bool, int, uint32) ([]segments.ListItem, bool, error)
+
+type segmentStub struct {
```

Comment:
segmentStub could probably go in a different file, then later you could have a single file for just the meta tests, a single file for the put tests, etc.